bundles/org.openhab.binding.sonnen/src/main/java/org/openhab/binding/sonnen/internal/SonnenBindingConstants.java

```diff
@@ -45,4 +45,8 @@ public class SonnenBindingConstants {
     public static final String CHANNELFLOWGRIDBATTERYSTATE = "flowGridBatteryState";
     public static final String CHANNELFLOWPRODUCTIONBATTERYSTATE = "flowProductionBatteryState";
     public static final String CHANNELFLOWPRODUCTIONGRIDSTATE = "flowProductionGridState";
+
+    // List of new Channel ids for PowerMeter API
+    public static final String KWHIMPORTEDSTATE = "energyImportedState";
```

Comment:
These channel ids are now inconsistent with the actual channel ids defined in `thing-types.xml`, so updating the channels will not work. I would propose to rename the constants as well (e.g. `ENERGY_IMPORTED_STATE`) and do the corresponding changes in `thing-types.xml`.

Reply:
Agree